stable_baselines/common/base_class.py

```diff
@@ -457,7 +477,67 @@ def _save_to_file(save_path, data=None, params=None):
             cloudpickle.dump((data, params), save_path)
 
     @staticmethod
-    def _load_from_file(load_path):
+    def _save_to_file_zip(save_path, data=None, params=None):
+        """Save model to a .zip archive
+
+        :param save_path: (str or file-like) Where to store the model
+        :param data: (OrderedDict) Class parameters being stored
+        :param params: (OrderedDict) Model parameters being stored
+        """
+        # data/params can be None, so do not
+        # try to serialize them blindly
+        if data is not None:
+            serialized_data = data_to_json(data)
+        if params is not None:
+            serialized_params = params_to_bytes(params)
+            # We also have to store list of the parameters
+            # to store the ordering for OrderedDict.
+            # We can trust these to be strings as they
+            # are taken from the Tensorflow graph.
+            serialized_param_list = json.dumps(
+                list(params.keys()),
+                indent=4
+            )
+
+        # Check postfix if save_path is a string
+        if isinstance(save_path, str):
+            _, ext = os.path.splitext(save_path)
+            if ext == "":
+                save_path += ".zip"
+
+        # Create a zip-archive and write our objects
+        # there. This works when save_path
+        # is either str or a file-like
+        with zipfile.ZipFile(save_path, "w") as file:
```

Comment:
`file` is reserved name in python (python 2 in fact...), please use `file_handler` or `file_`

Reply:
Ah right. I was wondering why syntax highlight did not detect it / everything worked fine, but that was the thing I remembered.